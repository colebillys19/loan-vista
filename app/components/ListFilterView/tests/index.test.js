// import React from 'react';
// import { render } from 'react-testing-library';
// import { MuiPickersUtilsProvider } from '@material-ui/pickers';
// import MomentUtils from '@date-io/moment';

// import ListFilter from '../index';

// const mockProps = {
//   dispatchFetchData: jest.fn(),
//   fetchParams: {
//     currentTotal: '80',
//     dateFrom: '',
//     dateTo: '',
//     keyword: '',
//     sortCol: 'date',
//     sortOrder: 'desc',
//   },
// };

// const component = (
//   <MuiPickersUtilsProvider utils={MomentUtils}>
//     <ListFilter {...mockProps} />
//   </MuiPickersUtilsProvider>
// );

describe('<ListFilter />', () => {
  // it('Expect to not log errors in console', () => {
  //   const spy = jest.spyOn(global.console, 'error');
  //   render(component);

  //   expect(spy).not.toHaveBeenCalled();
  // });

  // it('Should render and match the snapshot', () => {
  //   const {
  //     container: { firstChild },
  //   } = render(Component);
  //   expect(firstChild).toMatchSnapshot();
  // });

  it('', () => {
    expect(false).toBeTruthy();
  });
});
