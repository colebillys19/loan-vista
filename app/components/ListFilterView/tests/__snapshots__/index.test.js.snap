// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListFilter /> Should render and match the snapshot 1`] = `
<form
  class="styledComponents__StyledForm-sc-4etz7c-0 GpOgS"
>
  <span
    class="styledComponents__FieldWrapper-at5kq4-0 heWTRq"
  >
    <label
      class="styledComponents__StyledLabel-at5kq4-3 iOyTto"
      for=""
    >
      from
    </label>
    <span
      class="styledComponents__DatePickerWrapper-sc-18al4hg-1 exTZeK"
    >
      <div
        class="MuiFormControl-root MuiTextField-root styledComponents__StyledKeyboardDatePicker-sc-18al4hg-2 dzqpqB styledComponents__StyledTextField-sc-18al4hg-3 hcJBFK"
        title=""
      >
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedEnd"
            placeholder="date"
            type="text"
            value=""
          />
          <div
            class="MuiInputAdornment-root MuiInputAdornment-positionEnd"
          >
            <button
              class="MuiButtonBase-root MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        
      </div>
      <div
        class="styledComponents__ErrorIndicator-sc-18al4hg-0 bHXort"
      />
    </span>
  </span>
  <span
    class="styledComponents__FieldWrapper-at5kq4-0 heWTRq"
  >
    <label
      class="styledComponents__StyledLabel-at5kq4-3 iOyTto"
      for=""
    >
      to
    </label>
    <span
      class="styledComponents__DatePickerWrapper-sc-18al4hg-1 exTZeK"
    >
      <div
        class="MuiFormControl-root MuiTextField-root styledComponents__StyledKeyboardDatePicker-sc-18al4hg-2 dzqpqB styledComponents__StyledTextField-sc-18al4hg-3 hcJBFK"
        title=""
      >
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedEnd"
            placeholder="date"
            type="text"
            value=""
          />
          <div
            class="MuiInputAdornment-root MuiInputAdornment-positionEnd"
          >
            <button
              class="MuiButtonBase-root MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        
      </div>
      <div
        class="styledComponents__ErrorIndicator-sc-18al4hg-0 bHXort"
      />
    </span>
  </span>
  <span
    class="styledComponents__FieldWrapper-at5kq4-0 heWTRq"
  >
    <label
      class="styledComponents__StyledLabel-at5kq4-3 iOyTto"
      for=""
    >
      or
    </label>
    <div
      class="MuiInputBase-root MuiInput-root MuiInput-underline styledComponents__StyledSelect-tuipg9-1 evfkEJ"
    >
      <div
        aria-haspopup="listbox"
        class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input"
        role="button"
        tabindex="0"
      >
        date range
      </div>
      <input
        aria-hidden="true"
        class="MuiSelect-nativeInput"
        tabindex="-1"
        value="0"
      />
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSelect-icon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M7 10l5 5 5-5z"
        />
      </svg>
    </div>
  </span>
  <span
    class="styledComponents__FieldWrapper-at5kq4-0 heWTRq"
  >
    <label
      class="styledComponents__StyledLabel-at5kq4-3 iOyTto"
      for=""
    >
      and/or
    </label>
    <div
      class="MuiFormControl-root MuiTextField-root styledComponents__StyledTextField-sc-12js232-0 loVvUQ"
    >
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiInput-input"
          placeholder="keyword search"
          type="text"
          value=""
        />
      </div>
    </div>
  </span>
  <span
    class="styledComponents__FieldWrapper-at5kq4-0 heWTRq"
  >
    <button
      aria-label="refresh"
      class="MuiButtonBase-root MuiIconButton-root styledComponents__StyledIconButton-p72z6f-0 fSMSXL"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiIconButton-label"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root"
          focusable="false"
          style="height: 2.5rem; width: 2.5rem;"
          viewBox="0 0 24 24"
        >
          <path
            d="M0 0h24v24H0V0z"
            fill="transparent"
          />
          <path
            d="M17.65 6.35c-1.63-1.63-3.94-2.57-6.48-2.31-3.67.37-6.69 3.35-7.1 7.02C3.52 15.91 7.27 20 12 20c3.19 0 5.93-1.87 7.21-4.56.32-.67-.16-1.44-.9-1.44-.37 0-.72.2-.88.53-1.13 2.43-3.84 3.97-6.8 3.31-2.22-.49-4.01-2.3-4.48-4.52C5.31 9.44 8.26 6 12 6c1.66 0 3.14.69 4.22 1.78l-1.51 1.51c-.63.63-.19 1.71.7 1.71H19c.55 0 1-.45 1-1V6.41c0-.89-1.08-1.34-1.71-.71l-.64.65z"
            fill="rgb(80, 103, 55)"
          />
        </svg>
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </span>
  <span
    class="styledComponents__FieldWrapper-at5kq4-0 heWTRq"
  >
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text styledComponents__StyledButton-sc-1jxtcmc-0 kLtwHh styledComponents__StyledButton-sc-86ulmm-0 bbadGD"
      tabindex="0"
      type="submit"
    >
      <span
        class="MuiButton-label"
      >
        find
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </span>
</form>
`;
